File: c_rust_day_7/postgres_function.c
#include<stdio.h>

int sum(int a, int b); // function prototype

int
main() {

  int a = 10, b = 20, s;

  s  = sum(a,b);
  
  printf("%d", s);
  
  return 0;
}

int sum(int a, int b) {

	return a+b; // returning a value from function
    
}

File: c_rust_day_7/postgres_function.rs
fn main() {

	let a: i32 = 10;
    let b: i32 = 20;
    let c: i32;
    
    c = sum(a, b);
    
    println!("{}", c);

}

fn sum(a: i32, b: i32) -> i32 {

	a+b // returning value from function
    
}
